@xen-orchestra/lite/index.html

```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="UTF-8" />
     <link rel="icon" href="/favicon.ico" />
+    <link rel="manifest" href="/manifest.json">
```

Comment:
Although `.json` extension is supported, it would be preferable to use the `.webmanifest` [recommended file extension](https://www.w3.org/TR/appmanifest/#media-type-registration)

```suggestion
    <link rel="manifest" href="/manifest.webmanifest">
```

Reply:
Have already made that change.